### ISSUE TITLE
ComputeImage image-from-url-raw sample broken

### ISSUE THREAD

**Issue by trodge** [GoogleCloudPlatform/k8s-config-connector#269]
The sample [here](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/samples/resources/computeimage/image-from-url-raw/compute_v1beta1_computeimage.yaml) contains a value for `licenses` which causes the following error:
```
  Warning  UpdateFailed  14m (x12 over 14m)    computeimage-controller  Update call failed: error applying desired state: Error creating Image: googleapi: Error 404: The resource 'projects/tjr-dm-test-1/global/licenses/licensing' was not found, notFound
```

**Comment by maqiuyujoyce** [comment]
Hi @trodge , thank you for reporting the issue. We've successfully reproduced it and will look into it soon.

**Comment by caieo** [comment]
This is now fixed in release 1.22.0 ([notes here](https://github.com/GoogleCloudPlatform/k8s-config-connector/releases/tag/1.22.0)).
